.github/workflows/ci.yml

```diff
@@ -38,14 +40,57 @@ jobs:
             linode/linode-blockstorage-csi-driver
           tags: |
             type=raw,value=latest,enable=${{ github.ref == format('refs/heads/{0}', 'main') }}
+            type=raw,value=${{ github.head_ref || github.ref_name }} ,enable=${{ github.ref != format('refs/heads/{0}', 'main') }}
```

Comment:
I wonder if we should also delete the pushed image once tests are done. Otherwise, we'll have too many images just pushed to dockerhub.

Reply:
I was originally doing that but Evan said it was okay to leave it there

Reply:
:thinking: What if we used Docker Hub for releases only, and GHCR for releases and testing images?

Reply:
Yeah, I would prefer not pushing images to official linode dockerhub when they are just test images. GHCR sounds good to me for testing.